### PR TITLE
HDDS-2213. Reduce key provider loading log level in OzoneFileSystem#ge…

### PR DESCRIPTION
**Pull request by shwetayakkali**
…tAdditionalTokenIssuers

Updated Log level to debug.

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 2848 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| -1 | mvninstall | 44 | hadoop-hdds in trunk failed. |
| -1 | mvninstall | 41 | hadoop-ozone in trunk failed. |
| -1 | compile | 21 | hadoop-hdds in trunk failed. |
| -1 | compile | 15 | hadoop-ozone in trunk failed. |
| +1 | checkstyle | 63 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 873 | branch has no errors when building and testing our client artifacts. |
| -1 | javadoc | 23 | hadoop-hdds in trunk failed. |
| -1 | javadoc | 19 | hadoop-ozone in trunk failed. |
| 0 | spotbugs | 973 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| -1 | findbugs | 32 | hadoop-hdds in trunk failed. |
| -1 | findbugs | 21 | hadoop-ozone in trunk failed. |
||| _ Patch Compile Tests _ |
| -1 | mvninstall | 35 | hadoop-hdds in the patch failed. |
| -1 | mvninstall | 37 | hadoop-ozone in the patch failed. |
| -1 | compile | 25 | hadoop-hdds in the patch failed. |
| -1 | compile | 18 | hadoop-ozone in the patch failed. |
| -1 | javac | 25 | hadoop-hdds in the patch failed. |
| -1 | javac | 18 | hadoop-ozone in the patch failed. |
| +1 | checkstyle | 60 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 729 | patch has no errors when building and testing our client artifacts. |
| -1 | javadoc | 22 | hadoop-hdds in the patch failed. |
| -1 | javadoc | 18 | hadoop-ozone in the patch failed. |
| -1 | findbugs | 32 | hadoop-hdds in the patch failed. |
| -1 | findbugs | 18 | hadoop-ozone in the patch failed. |
||| _ Other Tests _ |
| -1 | unit | 28 | hadoop-hdds in the patch failed. |
| -1 | unit | 27 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 32 | The patch does not generate ASF License warnings. |
| | | 5214 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1556 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle |
| uname | Linux 22e717934409 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 137546a |
| Default Java | 1.8.0_222 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/branch-mvninstall-hadoop-hdds.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/branch-mvninstall-hadoop-ozone.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/branch-compile-hadoop-hdds.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/branch-compile-hadoop-ozone.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/branch-javadoc-hadoop-hdds.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/branch-javadoc-hadoop-ozone.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/branch-findbugs-hadoop-hdds.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/branch-findbugs-hadoop-ozone.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-mvninstall-hadoop-hdds.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-mvninstall-hadoop-ozone.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-compile-hadoop-hdds.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-compile-hadoop-ozone.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-compile-hadoop-hdds.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-compile-hadoop-ozone.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-javadoc-hadoop-hdds.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-javadoc-hadoop-ozone.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-findbugs-hadoop-hdds.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-findbugs-hadoop-ozone.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/testReport/ |
| Max. process+thread count | 446 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/ozonefs U: hadoop-ozone/ozonefs |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1556/1/console |
| versions | git=2.7.4 maven=3.3.9 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by avijayanhwx** [comment]
+1 LGTM

File: hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OzoneFileSystem.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ public URI getKeyProviderUri() throws IOException {
     try {
       keyProvider = getKeyProvider();
     } catch (IOException ioe) {
-      LOG.error("Error retrieving KeyProvider.", ioe);
+      LOG.debug("Error retrieving KeyProvider.", ioe);
```
Comment (arp7) [inline hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/OzoneFileSystem.java]: You can also update the text to say something like "this error is benign when no keyprovider is configured."

**Pull request merged**